cpp/models/abm/npi.cpp

```diff
@@ -0,0 +1,149 @@
+/* 
+* Copyright (C) 2020-2023 German Aerospace Center (DLR-SC)
+*
+* Authors: David Kerkman
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "abm/npi.h"
+#include "abm/person.h"
+#include "abm/location.h"
+
+namespace mio
+{
+namespace abm
+{
+
+struct AcceptBaseStringency : AcceptBase {
```

Comment:
Could you provide some comments on this struct on how / what it is meant to be (used)?

Reply:
I have done so.